integration-tests/src/test/java/uk/gov/di/authentication/api/CheckUserExistsIntegrationTest.java

```diff
@@ -85,6 +86,9 @@ void shouldCallUserExistsEndpointAndReturnAuthenticationRequestStateWhenUserExis
         assertThat(checkUserExistsResponse.getEmail(), equalTo(emailAddress));
         assertThat(checkUserExistsResponse.getMfaMethodType(), equalTo(mfaMethodType));
         assertTrue(checkUserExistsResponse.doesUserExist());
+        if (MFAMethodType.SMS.equals(mfaMethodType)) {
+            assertThat(checkUserExistsResponse.getPhoneNumberLastThree(), equalTo("321"));
+        }
```

Comment:
I suggest adding another if statement to check if the mfaMethodType is of type AUTH_APP. If so, then to check if the getPhoneNumberLastThree() returns an empty string.